lib/rev-api/models/order_request.rb

```diff
@@ -169,6 +194,50 @@ class Input < ApiSerializable
 
     # External URL, if sources wasn't POSTed as input (YouTube, Vimeo, Dropbox, etc)
     attr_reader :external_link
+
+    #Optional, list of glossary entries.
+    attr_reader :glossary
+
+    #Optional, list of speaker names.
+    attr_reader :speaker_names
+
+    # Optional, list of accents.
+    attr_reader :accents
+
+    SUPPORTED_ACCENTS = {
+      :american_neutral => 'AmericanNeutral',
+      :american_southern => 'AmericanSouthern',
+      :asian => 'Asian',
+      :australian => 'Australian',
+      :british => 'British',
+      :indian => 'Indian',
+      :other => 'Other',
+      :unknown => 'Unknown'
+    }
+
+    def validate_glossary
+      if !glossary.nil?
+        raise(ArgumentError, "Glossary must not exceed #{GLOSSARY_ENTRIES_LIMIT} entries") if glossary.length > GLOSSARY_ENTRIES_LIMIT
+        glossary.each { |term|
+          raise(ArgumentError, "Glossary entries cannot exceed #{GLOSSARY_ENTRY_LENGTH_LIMIT} characters") if term.length > GLOSSARY_ENTRY_LENGTH_LIMIT
+        }
+      end
+    end
+
+    def validate_speaker_names
+      if !speaker_names.nil?
```

Comment:
You probably can use something like `if speaker_names ....`, instead of negating `nil?` call.
I remember that Rails have helper `.present?`, but not sure if there is something like it in standard library

Reply:
Yeah, .present was my first instinct as well. Couldn't find an equivalent for plain ruby unfortunately. I'll remove the !nil call like you suggest.